app/src/main/java/com/example/marvelapp/adapter/CharacterAdapter.kt

```diff
@@ -0,0 +1,38 @@
+package com.example.marvelapp.adapter
+
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.example.marvelapp.R
+import com.example.marvelapp.databinding.CardviewCharacterBinding
+import com.example.marvelapp.entity.Character
+
+class CharacterAdapter(private val characters: List<Character>) :
+    RecyclerView.Adapter<CharacterAdapter.ViewHolder>() {
+
+    override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
+        return ViewHolder(
+            LayoutInflater.from(parent.context).inflate(R.layout.cardview_character, parent, false)
+        )
+    }
+
+    override fun onBindViewHolder(holder: ViewHolder, position: Int) {
+        holder.bind(characters[position])
+    }
+
+    override fun getItemCount(): Int = characters.size
+
+    class ViewHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {
+        private val binding = CardviewCharacterBinding.bind(itemView)
+        fun bind(item: Character) {
+            item.let {
+                binding.apply {
+                    Glide.with(itemView.context).load(item.img).into(this.cardviewImage);
```

Comment:
please put each call in a separated line

Reply:
and remove de ;

Reply:
done